qdrant-landing/content/documentation/fastembed/fastembed-colbert.md

```diff
@@ -0,0 +1,146 @@
+
+With FastEmbed, you can use ColBERT to generate multivector embeddings. ColBERT is a powerful retrieval model that combines the strength of BERT embeddings with efficient late interaction techniques. FastEmbed will provide you with an optimized pipeline to utilize these embeddings in your search tasks.
+
+Please note that ColBERT requires more resources than other no-interaction models. We recommend you use ColBERT as a re-ranker instead of a first-stage retriever. 
+
+The first-stage retriever can retrieve 100-500 examples. This task would be done by a simpler model. Then, you can rank the leftover results with ColBERT.
+
+## Setup
+
+This command imports all late interaction models for text embedding.
+
+```python
+from fastembed import LateInteractionTextEmbedding
+```
+You can list which models are supported in your version of FastEmbed.
+
+```python
+LateInteractionTextEmbedding.list_supported_models()
+```
+This command displays the available models. The output shows details about the ColBERT model, including its dimensions, description, size, sources, and model file.
+
+```python
+[{'model': 'colbert-ir/colbertv2.0',
+  'dim': 128,
+  'description': 'Late interaction model',
+  'size_in_GB': 0.44,
+  'sources': {'hf': 'colbert-ir/colbertv2.0'},
+  'model_file': 'model.onnx'}]
+```
+Now, load the model.
+
+```python
+embedding_model = LateInteractionTextEmbedding("colbert-ir/colbertv2.0")
+```
+The model files will be fetched and downloaded, with progress showing.
+
+## Embed data
+
+First, you need to define both documents and queries.
+
+```python
+documents = [
+    "ColBERT is a late interaction text embedding model, however, there are also other models such as TwinBERT.",
+    "On the contrary to the late interaction models, the early interaction models contains interaction steps at embedding generation process",
+]
+queries = [
+    "Are there any other late interaction text embedding models except ColBERT?",
+    "What is the difference between late interaction and early interaction text embedding models?",
+]
+```
+
+**Note:** ColBERT computes document and query embeddings differently. Make sure to use the corresponding methods.
+
+Now, create embeddings from both documents and queries.
+
+```python
+document_embeddings = list(
+    embedding_model.embed(documents)
+)  # embed and qury_embed return generators,
+# which we need to evaluate by writing them to a list
+query_embeddings = list(embedding_model.query_embed(queries))
+
+```
+Display the shapes of document and query embeddings.
+
+```python
+document_embeddings[0].shape, query_embeddings[0].shape
+```
+
+You should get something like this:
+
+```python
+((26, 128), (32, 128))
+```
+
+Don't worry about query embeddings having the bigger shape in this case. ColBERT authors recommend to pad queries with [MASK] tokens to 32 tokens. They also recommends to truncate queries to 32 tokens, however we don't do that in FastEmbed, so you can put some straight into the queries.
```

Comment:
```suggestion
Don't worry about query embeddings having the bigger shape in this case. ColBERT authors recommend to pad queries with [MASK] tokens to 32 tokens. They also recommend truncating queries to 32 tokens, however, we don't do that in FastEmbed so that you can put some straight into the queries.
```